pkg/api/server/register_containers.go

```diff
@@ -1319,7 +1410,95 @@ func (s *APIServer) registerContainersHandlers(r *mux.Router) error {
 	// tags:
 	//   - containers
 	// summary: Attach to a container
-	// description: Hijacks the connection to forward the container's standard streams to the client.
+	// description: |
+	//   Attach to a container to read its output or send it input. You can attach
```

Comment:
Maybe just mention that the format is identical to Compat Attach?

Reply:
Properly better to do it the other way around? I mean I expect only a small number of people to looks at out compat docs. I would assume most people would use the docker docs or the libpod specific endpoint docs. SO having it here sounds better to me.